jobs/mobile-developer.md

```diff
@@ -0,0 +1,60 @@
+# Mobile Developer (Android/iOS)
+
+We are looking for experienced and initiative Mobile Developers to grow our team and develop meaningful products together. We value openness and 
+knowledge sharing, believe that communication is a key and develop the remote culture for around 6 years. We would like to work with someone who shares our values,
+acts independently and proactively.
+
+
+## How the job looks
+
+- We have jobs both for Android and iOS developers, experienced (or ready to improve their skills) in React Native. We work with Kotlin + RxJava + Dagger for Android
+and Swift + RxSwift for iOS. We also use React Native and going to use it more
+- You will work with a team of developers (3-5 people) on the current project developing new features
+- Usually our projects last for about 1,5-2 years (but it could be more/less). Our developers don't work on more than 1 project at a time
+- Salary 𝚫: 24k—48k USD
+- For the development process we have Kanban desks, communication in channels and calls, including direct communication with clients. Tools that help us are
+Slack, Trello, Notion. We also us Holacracy framework for organizational governance
+- Apart from working on projects we will encourage you to gain and share knowledge, discuss your personal development and contribute to company's activities
+- You'll have colleagues all around the globe, variety of channels in Slack like #animal-therapy, #cooking, #ihateremotework and remote team spirit
+
+## How a good candidate looks
+
+- You have solid experience in anything we use - Kotlin, Swift or React Native
+- You want to work with React Native, even if you have not much experience with it
+- ***(? tech stuff- like more details about frameworks and technologies we would like to see in background ?)***
+- Your English skills allow you to feel comfortable communicating in it (verbally and written) the whole working day
+- You know how important it is to communicate with the team and inform others about what you are doing, so you stick to open and frequent communication
+- Learning process is an important part of your professional activities
+- Teamwork is not an abstract word for you, but a set of principles and agreements you follow
+- You value the quality of the code 
```

Comment:
I think we should formulate it better, but not sure I can make it properly. Any ideas?